setup/verify_setup.robot

```diff
@@ -5,6 +5,7 @@ Library    SeleniumLibrary
 *** Test Cases ***
 
 Test Open Browser
-    Open Browser    https://www.google.com    browser=${BROWSER}
-    Sleep    10s 
+    Open Browser    http://localhost:7272    browser=${BROWSER}
+    Sleep    5s
```

Comment:
I've kept the 10 seconds since if this first time then in Windows it can ask the firewall rules and if the browser is closed without giving the permissions then it can be tricky to get those firewall permissions back

Reply:
Sounds reasonable. Reverted back to 10 seconds.